tutorials/mysql_setup.md

```diff
@@ -47,3 +47,53 @@ you can remove that.
 GRANT ALL PRIVILEGES ON panel.* TO 'pterodactyl'@'127.0.0.1' WITH GRANT OPTION;
 FLUSH PRIVILEGES;
 ```
+:::tip
+The following is to add a database managed by the panel for it's game servers
+:::
+## Adding Database to Pterodactyl Panel
+In this guide we assume you have successfully installed the panel using the specified MariaDB as your database and it is on the `0.7.15` version.
+## Adding DB user
+First of all you will need to add a user to your database.This is simply done by connecting to your database via ssh and running the following. And entering your database root password, if you haven't set one just hit enter.
+```bash
+mysql -u root -p
+```
+Once you are in the database you need to change the db schema by running
+```sql
+USE mysql;
+```
+To than create a user run.<span style="color:red"> Change`data` and `password` to keep your database safe! </span>
+```sql
+CREATE USER 'data'@'127.0.0.1' IDENTIFIED BY 'password';
+```
+Than add privileges for the user.
+:::danger
+If your database is on a different host than your panel/daemon make sure to use `%` instead of `127.0.0.1` bellow
```

Comment:
```suggestion
If your database is on a different host than the one where your panel(daemon is installed, make sure to use `%` instead of `127.0.0.1` below
```